sep53/src/lib.rs

```diff
@@ -0,0 +1,162 @@
+use defuse_crypto::{CryptoHash, Curve, Ed25519, Payload, SignedPayload, serde::AsCurve};
+use impl_tools::autoimpl;
+use near_sdk::{env, near};
+use serde_with::serde_as;
+
+/// See [SEP-53](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0053.md)
+#[near(serializers = [json])]
+#[serde(rename_all = "camelCase")]
```

Comment:
We use `snake_case` in other formats, let's keep this convention here, too
```suggestion
#[serde(rename_all = "snake_case")]
```